index.html

```diff
@@ -5,23 +5,85 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>To-Dos App</title>
 </head>
-  <body>
-      <h1>To-Dos</h1>
-      <form>
-        <input type="text" placeholder="Enter a task">
-        <input type="submit" value="+">
-      </form>
-      <form>
-        <input type="checkbox" id="checkbox1" value="dummy1" checked>
-        <label for="checkbox1"><s>Completed Task</s></label><br>
-        <input type="checkbox" id="checkbox2" value="dummy2">
-        <label for="checkbox2">Incomplete Task</label>
-        <button type="submit">Submit Response</button>
-      </form>
-      <menu>
-        <li><button>All Tasks</button></li>
-        <li><button>Complete Tasks</button></li>
-        <li><button>Incomplete Tasks</button></li>
-      </menu>
-  </body>
+<body>
+  <header>
+    <h1>To-Dos
+    </h1>
```

Comment:
Since `<h1>` does not have any child elements other than a short text, it should remain as a single line, like so: `<h1>To-Dos</h1>`